SpringProject/src/main/java/com/gradproject2019/users/exception/CredentialsExistException.java

```diff
@@ -0,0 +1,8 @@
+package com.gradproject2019.users.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.CONFLICT, reason="Credentials are already in use.")
+public class CredentialsExistException extends RuntimeException {
```

Comment:
I think I would replace 'credentials' with 'userInfo' or 'username or email', otherwise it seems like I can guess a user's username and password by trying to register one with the same credentials and getting this exception.